ui-ngx/src/app/shared/models/rule-node.models.ts

```diff
@@ -373,6 +373,10 @@ export enum MessageType {
   ATTRIBUTES_DELETED = 'ATTRIBUTES_DELETED',
   ALARM_ACKNOWLEDGED = 'ALARM_ACKNOWLEDGED',
   ALARM_CLEARED = 'ALARM_CLEARED',
+  ALARM_ASSIGNED = 'ALARM_ASSIGN',
```

Comment:
I don't like the diff between names: 

* 'ALARM_ASSIGN' but not 'ALARM_ASSIGNED'